File: src/new main/Integrated/prototype/Property.py
class Property(object):
    def __init__(self, name, specification, propertyType):
        self.name = name
        self.specification = specification
        self.type = propertyType

    def getValue(self):
        return None


class MCIProperty(Property):
    def __init__(self, name, specification, propertyType, etc):
        super(MCIProperty, self).__init__(name, specification, propertyType)
        self.etc = etc

    def getValue(self):
        return self.etc

File: src/new main/Integrated/src/new_simvasos/property/Property.java
package new_simvasos.property;

public class Property {

  String name;
  String specification;
  String type;
  
  public Property(String name, String specification, String propertyType) { //constructor
    this.name = name;
    this.specification = specification;
    this.type = propertyType;
  }

  // this method is for inherited classes' getValue call like MCI Property
  public double getValue() { return  -1; }
  public double getThresholdPatient() {return -1;}
}
